specs/notation-cli.md

```diff
@@ -7,15 +7,16 @@ This spec contains reference information on using notation commands. Each comman
 | Command                                     | Description                                                            |
 | ------------------------------------------- | ---------------------------------------------------------------------- |
 | [certificate](./commandline/certificate.md) | Manage certificates in trust store                                     |
-| [inspect](./commandline/inspect.md)         | Inspect signatures                                                     |
 | [key](./commandline/key.md)                 | Manage keys used for signing                                           |
-| [list](./commandline/list.md)               | List signatures of the signed artifact                                 |
-| [login](./commandline/login.md)             | Login to registries                                                    |
-| [logout](./commandline/logout.md)           | Log out from the logged in registries                                  |
+| [list](./commandline/list.md)               | List signatures of a signed OCI artifact                               |
+| [login](./commandline/login.md)             | Log into OCI registries                                                |
+| [logout](./commandline/logout.md)           | Log out from the logged in OCI registries                              |
 | [plugin](./commandline/plugin.md)           | Manage plugins                                                         |
-| [policy](./commandline/policy.md)           | Manage trust policy configuration for signature verification |
-| [sign](./commandline/sign.md)               | Sign artifacts                                                         |
-| [verify](./commandline/verify.md)           | Verify artifacts                                                       |
+| [policy](./commandline/policy.md)           | Manage trust policy configuration for signature verification           |
+| [sign](./commandline/sign.md)               | Sign OCI artifacts                                                     |
+| [verify](./commandline/verify.md)           | Verify OCI artifacts                                                   |
+| [inspect](./commandline/inspect.md)         | Inspect OCI signatures                                                 |
+| [blob](./commandline/blob.md)               | Sign, verify and inspect singatures associated with blobs                                |
```

Comment:
Same as below, these used to be sorted too, typo in "signatures", and last `|` does not line up.

Reply:
Fixed in the latest revision. Apologies for the churn. I somehow screwed up my local dev environment. I thought I fixed this in my earlier commit.